app/views/employers/plan_years/_publish_modal.html.erb

```diff
@@ -3,17 +3,30 @@
     <div class="modal-content">
       <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title">Confirm Publish Plan Year</h4>
+        <h4 class="modal-title">Document Upload Required to Proceed</h4>
       </div>
       <div class="modal-body">
         <p>
-          As submitted, this application is ineligible for coverage under the <%= site_short_name %> exchange for reasons identified below. Click Cancel to go back and correct inaccurate information. If the information provided is accurate, you may click Publish to proceed. However, if you choose to proceed and the eligibility determination is deemed correct by <%= site_short_name %> officials, this organization may not again apply for coverage for 90 days, or until the reason for ineligibility changes, whichever is later.
```

Comment:
this text needs to stay and be displayed conditionally.

per hannahs comment on the ticket:

Note: 17395 is a DIFFERENT pop-up modal that has a different trigger. We want to KEEP the modal and "force publish" functionality in 17395 (i.e. triggered when an ER is about to publish an ineligible application).

In THIS ticket, We want to REMOVE "force publish" functionality for an employer who has not yet uploaded a document. In other words, employers cannot publish a plan year without having uploaded a document. Employers CAN publish an ineligible application (per 17395).

-- 
the employer sees two different messages depending on whether they've uploaded documents or not. They're able to force publish if they've uploaded a document but invalid for another reason.